zebra-state/tests/basic.rs

```diff
@@ -0,0 +1,96 @@
+use color_eyre::eyre::Report;
+use color_eyre::eyre::{bail, ensure, eyre};
+use once_cell::sync::Lazy;
+use std::sync::{Arc, Once};
+use tempdir::TempDir;
+use tower::Service;
+use tracing_error::ErrorLayer;
+use tracing_subscriber::prelude::*;
+use tracing_subscriber::{fmt, EnvFilter};
+use zebra_chain::{
+    block::{Block, BlockHeaderHash},
+    serialization::ZcashDeserialize,
+};
+use zebra_test::transcript::Transcript;
+
+use zebra_state::*;
+
+static LOGGER_INIT: Once = Once::new();
+
+fn install_tracing() {
```

Comment:
Once this merges I'll fix the install_tracing stuff

Reply:
No need to block on it, I'm going to be afk for the next few hours so if you prefer to fix the install_tracing stuff now, i'm happy to rebase this PR on top of yours and de-conflict things